cmake/toolchain/zephyr/0.11/target.cmake

```diff
@@ -27,6 +27,10 @@ if("${ARCH}" STREQUAL "xtensa")
     set(SR_XT_TC_SOC intel_byt_adsp)
   elseif("${SOC_SERIES}" STREQUAL "broadwell_adsp")
     set(SR_XT_TC_SOC intel_bdw_adsp)
+  elseif("${SOC_SERIES}" STREQUAL "imx8")
+    set(SR_XT_TC_SOC nxp_imx_adsp)
```

Comment:
why isn't it `nxp_imx8_adsp`?

Reply:
The `zephyr-sdk-0.12.2/xtensa/` contains 2 toolchains: `nxp_imx_adsp` for imx8QM and imx8QXP and `nxp_imx8m_adsp` for imx8MP. 
So, the corresponding toolchain for `imx8` (which represents imx8QM and imx8QXP) is `nxp_imx_adsp`.

Reply:
That is my fault. In the past when I added the toolchain for i.MX8QM i named it imx instead of imx8. To make things 'better' the toolchain for imx8M Plus is indeed named imx8m.